packages/cozy-realtime/README.md

```diff
@@ -44,6 +44,37 @@ In order to listen to online and offline events, you should install the [Network
 
 ## Example
 
+### With CozyClient
+
+CozyRealtime comes with a plugin for CozyClient. This is the recommended way to use it: 
+```js
+//instanciation
+import { RealtimePlugin } from 'cozy-realtime'
+import { CozyClient} from 'cozy-client'
+
+const client = new CozyClient({})
+
+client.registerPlugin(RealtimePlugin)
+
+//usage
+
+this.realtime = client.plugins.realtime
+// To subscribe
+await this.realtime.subscribe('created', type, handleCreate)
+await this.realtime.subscribe('updated', type, handleCreate)
```

Comment:
I think it's a typo since handleCreate is repeated here for the "updated" event. Same in the "unsubscribe" part below.